reporting-anything/src/download_report.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import _ from 'lodash';
+import { ArgumentParser } from 'argparse';
+import { exit } from 'process';
+import { downloadVisualReport } from './download_helpers.js';
+import { REPORT_TYPE } from './constants.js';
+
+// url OR report source type and ID
+const parser = new ArgumentParser({
+  description: 'Argument parser for Reporting Anything'
+});
+
+parser.add_argument('-u', '--url', {help: 'url of report'});
+parser.add_argument('-t', '--type', {help: 'report source type'});
+parser.add_argument('-id', '--object-id', {help: 'saved object ID'});
```

Comment:
use one character for short opts, maybe `-o`? `-id` usually equals to `-i -d` by convention